core/server/worker/src/main/java/alluxio/worker/grpc/DoraWorkerClientServiceHandler.java

```diff
@@ -118,7 +120,10 @@ public void listStatus(ListStatusPRequest request,
           request.getOptions().hasRecursive() ? request.getOptions().getRecursive() : false);
       UfsStatus[] statuses = mWorker.listStatus(request.getPath(), options);
       if (statuses == null) {
-        responseObserver.onError(new FileNotFoundException("File or Directory Not Found"));
+        responseObserver.onError(
+            new AlluxioRuntimeException(Status.NOT_FOUND,
+                String.format("%s Not Found", request.getPath()),
+                null, ErrorType.Internal, false).toGrpcStatusException());
```

Comment:
@jja725 do you think `ErrorType.Internal` is the proper error type?

Reply:
I feel like this is more of a user error in this case

Reply:
OK. Changed to ErrorType.User
Thank you, @jja725